lib/modules/manager/runtime-version/extract.ts

```diff
@@ -0,0 +1,25 @@
+import { regEx } from '../../../util/regex';
+import { DockerDatasource } from '../../datasource/docker';
+import type { PackageDependency, PackageFileContent } from '../types';
+
+export const pythonRuntimeRegex = regEx(
+  '^python-(?<version>\\d+\\.\\d+\\.\\d+)$',
+);
+
+export function extractPackageFile(content: string): PackageFileContent | null {
+  const regexResult = pythonRuntimeRegex.exec(content);
+  const runtimeVersion = regexResult?.groups?.version;
+
+  if (runtimeVersion) {
+    const dep: PackageDependency = {
+      depName: 'python',
+      commitMessageTopic: 'Runtime Version',
```

Comment:
Why the specific message topic?

Reply:
Copied the structure from one of the existing managers.  Easily removed